bin/porta

```diff
@@ -830,7 +858,7 @@ module Porta
     end
 
     def run_apicast
-      system("#{container_tool} run -d --name apicast --rm -p 8080:8080 #{container_opts_local_hosts} -e THREESCALE_PORTAL_ENDPOINT=\"http://#{options[:apicast_access_token]}@#{container_internal_host}:3008/master/api/proxy/configs\" -e THREESCALE_DEPLOYMENT_ENV=staging -e BACKEND_ENDPOINT_OVERRIDE=\"http://#{container_internal_host}:3001\" #{options[:apicast_image]} >/dev/null && echo \"apicast\"")
+      system("#{container_tool} run -d --name apicast --rm -p 8080:8080 #{container_opts_local_hosts} -e THREESCALE_PORTAL_ENDPOINT=\"http://#{options[:apicast_access_token]}@#{container_internal_ip}:3008/master/api/proxy/configs\" -e THREESCALE_DEPLOYMENT_ENV=staging -e BACKEND_ENDPOINT_OVERRIDE=\"http://#{container_internal_host}:3001\" #{options[:apicast_image]} >/dev/null && echo \"apicast\"")
```

Comment:
```suggestion
      system("#{container_tool} run -d --name apicast --rm -p 8080:8080 #{container_opts_local_hosts} -e THREESCALE_PORTAL_ENDPOINT=\"http://#{options[:apicast_access_token]}@#{container_internal_ip}:3008/master/api/proxy/configs\" -e THREESCALE_DEPLOYMENT_ENV=staging -e BACKEND_ENDPOINT_OVERRIDE=\"http://#{container_internal_ip}:3001\" #{options[:apicast_image]} >/dev/null && echo \"apicast\"")
```

Reply:
Backend URL should also use an IP instead of host, because otherwise APIcast can't resolve it:
```
2024/06/12 08:28:00 [info] 14#14: *4 backend_client.lua:133: call_backend_transaction(): backend client uri: http://host.containers.internal:3001/transactions/authrep.xml?service_token=2c55d27da5d111e1b788f3493849f5a0d73fdf8eff12f0b99030e61a159076eb&service_id=2&usage%5Bhits%5D=1&user_key=whatever ok: false status: 0 body: nil error: host.containers.internal could not be resolved (3: Host not found), requestID=a65156f7706477f7911b2f6d20a99db3, client: 127.0.0.1, server: _, request: "GET /?user_key=whatever HTTP/1.1", host: "api-2.staging.apicast.dev.localhost:8080"
```